test/helpers/browser.js

```diff
@@ -7,6 +7,7 @@ var vm = require('vm');
 
 function createContext() {
   var context = {
+    self: this,
```

Comment:
> so users wouldnt have had this problem to begin with right?

Yes, I think so.

It's just a problem in our tests that are setting up a browser-like environment.